src/librbd/librbd.cc

```diff
@@ -519,6 +520,20 @@ namespace librbd {
     return 0;
   }
 
+  int RBD::features_to_string(uint64_t features, std::string *str_features)
+  {
+    *str_features = librbd::rbd_features_to_string(features, nullptr);
```

Comment:
Nit: it might be nice to return `-EINVAL` back if one or more parts fail to parse for these new functions.